docs/relations_guide.md

```diff
@@ -1,103 +1,123 @@
-# A guide to what relations to use where in the Cell Ontology
+# A guide to selecting **object properties** in the Cell Ontology (CL)
 
-## Recording location
 
-We record anatomical location by linking to terms from Uberon.
+### Recording anatomical location (general cell types)
 
-For most purposes we record the anatomical location of cells using **'part of'**. 
+To record anatomical locations for cell types, axioms are written using object properties to relate CL terms to Uberon terms.
 
-e.g. epithelial cell SubClassOf **'part of'** *some* epithelium 
+The most commonly used object property to record anatomical location is [**'part of'**](http://purl.obolibrary.org/obo/BFO_0000050). 
 
-means that
+For example, ['epithelial cell'](http://purl.obolibrary.org/obo/CL_0000066) has the following subclass:
 
- 1. All epithelial cells are part of an epithelium.  We wouldn't say 'epithelial cell' part_of some 'kidney tubule epithelium', because not all of them are.
+[**'part of'**](http://purl.obolibrary.org/obo/BFO_0000050) *some* [epithelium](http://purl.obolibrary.org/obo/UBERON_0000483) 
+
+In Manchester OWL syntax, this is represented as:
+
+SubClassOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000483))
+
+
+This statement connotes the following:
+
+ 1. All epithelial cells are part of an epithelium.
  1. All parts of an epithelial cell are part of an epithelium.
- 1. Epithelial cells are part_of some epithelium at all times.  This last stricture can be hard to apply in the context of development.  Some judgment may be required, e.g. -   (TBA)
+ 1. Epithelial cells are part of some epithelium at all times. This can be hard to apply in the context of development and may require additional consideration from an editor.
 
-Related Equivalence patterns: [cellPartOfAnatomicalEntity](https://github.com/obophenotype/cell-ontology/blob/master/src/patterns/dosdp-patterns/cellPartOfAnatomicalEntity.yaml).  Use this as a guide for creating simple grouping classes for cells of type X in structures of type Y.
```

Comment:
I think it is useful to associate related equivalence patterns throughout the text.